src/frontend/apps/e2e/__tests__/app-desk/mail-domain.spec.ts

```diff
@@ -137,7 +139,7 @@ test.describe('Mail domain', () => {
         'It seems that the page you are looking for does not exist or cannot be displayed correctly.',
       ),
     ).toBeVisible({
-      timeout: 15000,
+      timeout: 700000,
```

Comment:
I'm really worried about the consequences of allowing a worst-case duration of 15 minutes for our CI… I would either delete the test, or (preferably) fix the underlying problem.